tests/data/tsbContactTestFormProtectedB.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
Just a comment here to say that you did not need to create such a big form config for that specific test ;) It could have been a very minimal one.